src/test/components/TrackContextMenu.test.js

```diff
@@ -825,6 +826,84 @@ describe('timeline/TrackContextMenu', function () {
     );
   });
 
+  describe('show all local tracks in a process', function () {
+    function setupMoreTracks() {
+      const profile = getProfileWithMoreNiceTracks();
+      const store = storeWithProfile(profile);
+
+      render(
+        <Provider store={store}>
+          <TimelineTrackContextMenu />
+        </Provider>
+      );
+
+      function clickAllThreadPoolTracks() {
+        const threadPoolTracks = screen.getAllByText(/^ThreadPool#\d$/);
+        for (const track of threadPoolTracks) {
+          fireFullClick(track);
+        }
+      }
+
+      return {
+        ...store,
+        clickAllThreadPoolTracks,
+        profile,
+      };
+    }
+
+    // This runs 2 tests: the first right clicks a global track, the second
+    // right clicks the local track.
+    it.each([
+      { type: 'global', trackIndex: 0 },
+      {
+        type: 'local',
+        pid: 1000,
+        trackIndex: 0,
+      },
```

Comment:
Initially I wrote 2 tests, but they were nearly identical, and I thought it was more readable to group them this way, even though we don't do that usually... Tell me what you think.

Reply:
yeah, I thought that this way it's more obvious what difference the tests have. Otherwise it was a bit buried in the code and I couldn't find another way that I found better...